src/updater.ts

```diff
@@ -46,9 +53,7 @@ export class Updater {
     this.config = updaterConfig;
 
     this.fetcher = fetcher || new Fetcher(this.config.fetchTimeout);
-
-    // self._trusted_set = trusted_metadata_set.TrustedMetadataSet(data)
-    // self.config = config or UpdaterConfig()
+    this.config = config || updaterConfig;
```

Comment:
Can we do something like:

```typescript
this.config = { ...updaterConfig, ...config }
```
So that we end-up with the default config plus any value which have been overridden by the client.